showcase/src/app/home/home.component.html

```diff
@@ -1,100 +1,114 @@
-<h1 class="mat-display-2" translate>Stark documentation</h1>
+<div class="about-background">
+	<h1 class="about-title" [innerHTML]= "'SHOWCASE.HOMEPAGE.MAIN_TITLE' | translate"></h1>
+</div>
+<h2 class="mat-body-2" [innerHTML]="'SHOWCASE.HOMEPAGE.STARK_DESC_BEFORE_DETAIL' | translate">
+</h2>
+<h4 [innerHTML]="'SHOWCASE.HOMEPAGE.STARK_DESC_DETAIL' | translate"></h4>
```

Comment:
Can you use a `<p>` instead of `<h4>`?

Reply:
True `<p>` makes more sense, you can add the class `.mat-body-2` to create the same effect as h4. It's an alternative body font setting defined by Angular Material. You can also use the class `.mat-h4` on `<p>` if you want to keep the exact same style :-)